pkg/client/results/processing.go

```diff
@@ -67,25 +68,6 @@ type postProcessor func(string, string) (Item, error)
 // file only or all files with a given suffix (for instance).
 type fileSelector func(string, os.FileInfo) bool
 
-// Item is the central format for plugin results. Various plugin
-// types can be transformed into this simple format and set at a standard
-// location in our results tarball for simplified processing by any consumer.
-type Item struct {
```

Comment:
Had moved this into `types.go` when I was playing around with refactoring into subpackages; though that went nowhere. I can add it back so it removes a bunch of unnecessary diff.